pkger/models.go

```diff
@@ -2638,7 +2638,7 @@ func (q queries) influxDashQueries() []influxdb.DashboardQuery {
 			EditMode: "advanced",
 		}
 		// TODO: axe this builder configs when issue https://github.com/influxdata/influxdb/issues/15708 is fixed up
```

Comment:
anyone else curious about this foreboding line?

Reply:
Yup, definitely shows it to @gavincabbage this morning, especially since the issue was closed a couple days ago. I'd love to see @desa 's opinion on this